noxfile.py

```diff
@@ -136,6 +136,36 @@ def docs(session: Session) -> None:
     )
 
 
+@nox.session(python=python, venv_backend="conda")
+def schema_diagram(session: Session) -> None:
+    session.install("-e", "./jobmon_server")
+    outpath = Path(__file__).parent / "docsource" / "developers_guide" / "diagrams" / "erd.svg"
+    with tempfile.TemporaryDirectory() as tmpdir:
+        session.chdir(tmpdir)
+        session.run(
+            "jobmon_server", "init_db",
+            env={"JOBMON__DB__SQLALCHEMY_DATABASE_URI": "sqlite:///jobmon.db"}
```

Comment:
Interesting, so you need to initialize the database and load the schema to use this schemacrawler tool? 

Wonder if it'd be possible at all to infer directly from the source code (i.e. the sqlalchemy database model)